src/test/java/com/ui_init_setup/practiceproject/pages/SwagLabsLoginPage.java

```diff
@@ -53,8 +58,22 @@ private void setPassword(String password) {
         inputPassField.sendKeys(password);
     }
 
+    public String getErrorMessage() {
+        wdWait.until(ExpectedConditions.visibilityOfElementLocated(By.className("error-message-container")));
+        return errorMessage.getText();
+    }
+
+    public boolean errorMessIsShown() {
+        return errorMessage.isDisplayed();
+    }
+
     public SwagLabsInventoryPage loginWithCredentials(String username, String password) {
```

Comment:
these two methods should go before "logInAsValidUser" since they make calls to it (Clean Code: Vertical Ordering)